data/core/units/elves/Captain.cfg

```diff
@@ -22,7 +22,7 @@
     advances_to=Elvish Marshal
     cost=32
     usage=fighter
-    description= _ "Elves, unlike many other races, will quickly acknowledge and follow any of their peers who have experience in combat. In this they differ greatly from humanity, for whom leadership is often a matter of coercion and intimidation. The combination of their willingness to accept the wisdom of their leaders, and the tendency to choose leaders of merit is one of the quiet strengths of the elves."+{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}
+    description= _ "Elvish captains usually are the leaders of the patrols and garrisons that stand guard over the vast forests in elvish territory. Unlike leaders from other races, they are mostly proficient at navigating small skirmishes rather than large-scale battles, although they are capable of learning more detailed military tactics when necessary. Captains typically command small units that remain together for extended periods of time, allowing them to personally mentor and befriend their troops. They are often observed to lead through camaraderie and amicabilty, but still command great respect among their peers."+{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}
```

Comment:
I don't like "usually are the leaders".
How about "usually lead patrols and command garrisons"

Also, I prefer garrisons WHICH stand guard.

Reply:
https://www.grammarly.com/blog/which-vs-that/